workflows/rnaseq/rnaseq-core.wdl

```diff
@@ -30,6 +30,16 @@ workflow rnaseq_core {
         read_groups: "A string containing the read group information to output in the BAM file. If including multiple read group fields per-read group, they should be space delimited. Read groups should be comma separated, with a space on each side (i.e. ' , '). The ID field must come first for each read group and must be contained in the basename of a FASTQ file or pair of FASTQ files if Paired-End. Example: `ID:rg1 PU:flowcell1.lane1 SM:sample1 PL:illumina LB:sample1_lib1 , ID:rg2 PU:flowcell1.lane2 SM:sample1 PL:illumina LB:sample1_lib1`. These two read groups could be associated with the following four FASTQs: `sample1.rg1_R1.fastq,sample1.rg2_R1.fastq` and `sample1.rg1_R2.fastq,sample1.rg2_R2.fastq`"
         prefix: "Prefix for output files"
         contaminant_db: "A compressed reference database corresponding to the aligner chosen with `xenocp_aligner` for the contaminant genome"
+        align_sj_stitch_mismatch_n_max: {
+            description: "This overrides the STAR alignment default. Maximum number of mismatches for stitching of the splice junctions (-1: no limit) for: (1) non-canonical motifs, (2) GT/AG and CT/AC motif, (3) GC/AG and CT/GC motif, (4) AT/AC and GT/AT motif",
+            tool: "star",
+            tool_default: {
+                noncanonical_motifs: 0,
+                GT_AG_and_CT_AC_motif: -1,
```

Comment:
@a-frantz  - Sprocket also doesn't like this line because it doesn't understand the `-` thinks this is a string. I don't really want to quote it, but should we to get sprocket to pass and avoid missing other issues?

Reply:
Don't really like any option I see here, but I think the best we can do is quote the `-1` and leave a comment to the effect of `# TODO: remove quotes once sprocket parses negative numbers`